models/tokens/ethereum/tokens_ethereum_transfers_base.sql

```diff
@@ -0,0 +1,17 @@
+{{ config(
+        alias = alias('transfers_base'),
+        tags=['dunesql'],
+        partition_by = ['block_date'],
```

Comment:
using `block_date` instead of `block_month` here makes sense because we will have a lot of rows. :+1: